go.mod

```diff
@@ -4,12 +4,12 @@ go 1.17
 
 require (
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/go-logr/logr v0.4.0
```

Comment:
Updating this can commonly break the dependencies, which I think this has done, you'll probably need to pin these, we did this recently on Azure https://github.com/openshift/machine-api-provider-azure/blob/17e231c4fbc6eea84e171aa44b99bd7bfc83e58a/go.mod#L120